cn.apipost.apipost.metainfo.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id type="desktop">cn.apipost.apipost</id>
+  <name>apipost</name>
+  <summary>Apipost = Postman + Swagger + Mock + Jmeter, Backend, frontend, testing, and online collaboration with real-time content synchronization</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://wiki.apipost.cn/docs/introduction</project_license>
+  <content_rating type="oars-1.0" />
+  <developer_name>apipost developers</developer_name>
+  <description>
+    <p>
+      Apipost=Postman+Swagger+Mock+Jmeter backend, frontend, testing, and online collaboration with
+      real-time content synchronization
+    </p>
+    <p xml:lang="zh-Hans">
+      Apipost = Postman + Swagger + Mock + Jmeter 后端、前端、测试，同时在线协作，内容实时同步
+    </p>
```

Comment:
This is a repeat of the summary. You can drop this and instead translate the summary with `xml:lang`

Reply:
https://github.com/flathub/flathub/pull/5735/commits/bec72b3a007ed3f54c698ee897265068d6dae444